### DISCUSSION TITLE
⏳ Allow to query multiple sources at once in the Loop Grid

### DISCUSSION THREAD

**Discussion by e-cubed** [elementor/elementor#22998] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Elementor's loop grid currently only allows to source from a single type (Post, Page or Custom Post type)

### Describe the solution you'd like

It would be nice to to be able to query from multiple post types.

### Describe alternatives you've considered

We have tried to query by multiple taxonomies, but the taxonomies are still govern by the single source of post type. So this doesn't really solve the issue.

**Comment by createitweb** [comment]
I'm also looking for this feature.  I'd like to add multiple sources (advanced custom post types) for the Loop Grid query.

**Comment by OlliK1983** [comment]
I'm also looking for this feature. I'd like also to add multiple sources (advanced custom post types) for the Loop Grid query.

**Comment by lavendaniel** [comment]
Needed. Homepage news feed needs to show multiple different types of news from around the site. Very limiting to not be able to do this.

**Reply by andrewj1313** [reply]
I recently ran into this same issue with a client. We had to merge all of their post types into a single post type with numerous categories and it's causing a lot of confusion.

**Comment by anthonylemay** [comment]
Also needed! I really need this since i have some different custom posts using the same taxonomy / hierarchy that would need to be included in the same loop carousel feed.

**Comment by gallivale** [comment]
Any news on this?? Really a needed feature!

**Comment by SecurityDog88** [comment]
Need it too.

**Comment by jfagnonih4** [comment]
Need it too

**Comment by toineenzo** [comment]
For anybody that wants to be able to select multiple post types, take a look at my code snippet here:
https://codesnippets.cloud/snippet/toineenzo/PostsLoop-GridCarousel

**Reply by moonlightlizaa** [reply]
Thank you! This solved it for me

**Comment by Mart-0ne** [comment]
So the Elementor Team are just not answering to the threads ? Didn't know that the Elementor project was dead, but I guess it makes sense.
